app/lib/analyzer/pana_runner.dart

```diff
@@ -72,18 +74,24 @@ class _PanaRunner implements PanaRunner {
               PackageAnalyzer(toolEnv, urlChecker: _urlChecker);
           final isInternal = internalPackageNames.contains(package) ||
               packageStatus.isPublishedByDartDev;
-          return await analyzer.inspectPackage(
-            package,
-            version: version,
-            options: InspectOptions(
-              isInternal: isInternal,
-              pubHostedUrl: activeConfiguration.primaryApiUri.toString(),
-              analysisOptionsYaml: packageStatus.usesFlutter
-                  ? null
-                  : await getDefaultAnalysisOptionsYaml(),
-            ),
-            logger: Logger.detached('pana/$package/$version'),
-          );
+
+          Future<void> store(String fileName, Uint8List data) async {
+            final stream = () => Stream.value(data);
+            await imageStorage.upload(
+                package, version, stream, fileName, data.length);
+          }
+
+          return await analyzer.inspectPackage(package,
+              version: version,
+              options: InspectOptions(
+                isInternal: isInternal,
+                pubHostedUrl: activeConfiguration.primaryApiUri.toString(),
+                analysisOptionsYaml: packageStatus.usesFlutter
+                    ? null
+                    : await getDefaultAnalysisOptionsYaml(),
+              ),
+              logger: Logger.detached('pana/$package/$version'),
+              storeResource: store);
```

Comment:
Nit: `storeResource: store,` (`,` at the end)

Reply:
Done